### PR TITLE
Reference cli by general name

### PR DESCRIPTION
**Pull request by kotp**
This is mostly here to check the appveyor settings, but may be a good change anyway.

### CODE REVIEW

File: docs/TESTS.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+Exercises that have been fetched using the [Exercism command line client](http://www.exercism.io/cli) are delivered with a minimum of three files: a `readme.md` file, a `.dpr` file, and a `.pas` file.  The `.dpr` file is the Delphi project file and the `.pas` file is the test runner.  Load the Delphi project by either double clicking on the `.dpr` file or by opening the project from with in Delphi.  You will be responsible for creating a new `.pas` file that will contain your solution code that the tests will be run against.  Refer to the `readme.md` file for instruction on how to compile and execute your code.
```
Comment (rpottsoh) [inline docs/TESTS.md]: I see a typo, that has always been here.  `with in` should be `within`.

Comment (kotp) [inline docs/TESTS.md]: Nice, after you check the appveyor feel free to fix and push up, it will check.

**Comment by kotp** [comment]
I am unsure why appveyor is still registered for this project.

**Comment by rpottsoh** [comment]
@kytrinyx I think we need an assist.  We are not sure why Appveyor is still active for this project.

**Comment by kytrinyx** [comment]
To clarify: you want me to delete the appveyor project?

**Comment by rpottsoh** [comment]
Thanks for asking, good question.  We have no immediate plans to use Appveyor so would like for it not to process PRs.  We don't know what is involved in preventing Appveyor from attempting to process PRs.  We thought we had it disabled about 2 weeks ago but the other day it appeared again when a new PR was created.


Sent from my android device.

**Comment by kytrinyx** [comment]
You should be all set now.

**Comment by rpottsoh** [comment]
Thanks @kytrinyx all looks good.  What was the issue/trick?

**Comment by kytrinyx** [comment]
I deleted it :-) Not sure it got deleted or just paused or something in the previous round.

**Pull request merged**